File: LootMarket Backend/src/main/java/com/example/lootmarketbackend/dto/AstaDTO.java
package com.example.lootmarketbackend.dto;

public class AstaDTO {
    public int idAsta;
    public String emailCreatore;
    public String titolo;
    public String categoria;
    public double prezzoPartenza;
    public int anno;
    public int mese;
    public int giorno;
    public String descrizione;
    public double ultimaOfferta;
    public double sogliaMinima;
    public String tipoAsta;
    public Boolean offertaFatta;
    public String immagineAsta;

    public AstaDTO(int idAsta, String emailCreatore, String titolo, String categoria, double prezzoPartenza, int anno, int mese, int giorno, String descrizione, double ultimaOfferta, double sogliaMinima, String tipoAsta, Boolean offertaFatta, String immagineAsta) {
        this.idAsta = idAsta;
        this.emailCreatore = emailCreatore;
        this.titolo = titolo;
        this.categoria = categoria;
        this.prezzoPartenza = prezzoPartenza;
        this.anno = anno;
        this.mese = mese;
        this.giorno = giorno;
        this.descrizione = descrizione;
        this.ultimaOfferta = ultimaOfferta;
        this.sogliaMinima = sogliaMinima;
        this.tipoAsta = tipoAsta;
        this.offertaFatta = offertaFatta;
        this.immagineAsta = immagineAsta;
    }
}

File: LootMarket/app/src/main/java/com/danilo/lootmarket/Network/dto/AstaDTO.kt
package com.danilo.lootmarket.network.dto


data class AstaDTO(val idAsta: Int,
                   val emailCreatore: String,
                   val titolo: String ,
                   val categoria: String ,
                   val prezzoPartenza: Double,
                   val anno: Int,
                   val mese: Int,
                   val giorno: Int,
                   val descrizione: String,
                   val ultimaOfferta: Double,
                   val sogliaMinima: Double,
                   val tipoAsta: String,
                   var offertaFatta: Boolean,
                   val immagineAsta: String
)
